orange/src/main/java/com/hartwig/hmftools/orange/algo/OrangeAlgo.java

```diff
@@ -525,12 +523,14 @@ private static LilacSummaryData loadLilacData(@NotNull OrangeConfig config) thro
     @Nullable
     private static VirusInterpreterData loadVirusInterpreterData(@NotNull OrangeConfig config) throws IOException
     {
+        // TODO consider if the tumor only mode is better replaced with checking if WGS is configured
+        //  here and elsewhere below
```

Comment:
This is definately better. Best to rely on the source rather than some derived value. If there is a virus dir present, we can assume virus data can be loaded, unrelated to modes or ref/no-ref etc

Reply:
I think my TODO comment was already made obsolete by having changed the tumor only check to consider wgsRefConfig.